htdocs/core/class/html.form.class.php

```diff
@@ -10154,13 +10201,11 @@ public function selectInvoice($socid = -1, $selected = '', $htmlname = 'invoicei
 				$out .= '</select>';
 			}
 
-			print $out;
+			return $out;
 
 			$this->db->free($resql);
-			return $num;
 		} else {
 			dol_print_error($this->db);
-			return -1;
 		}
 	}
```

Comment:
Must add a return ''; 
so function will always return a string and change the doc of function from @return int into @return string